plugins/nodes/src/js/components/LeaderGrid.js

```diff
@@ -18,6 +19,14 @@ function timeFromNow(time) {
   return moment(time * 1000).fromNow();
 }
 
+function formattedDate(time) {
+  if (!time) {
+    return null;
+  }
+
+  return moment(time * 1000).format("MMMM Do YYYY, h:mm:ss a");
```

Comment:
The original design had no seconds. Do we really think it is valuable to show seconds?

Reply:
Actually, I think the format compatible with what Licensing does would be:
MMMM Do, YYYY h:mm a